lib/publiccloud/provider.pm

```diff
@@ -387,6 +387,12 @@ sub terraform_apply {
             $cmd .= sprintf(q(-var '%s=%s' ), $key, escape_single_quote($value));
         }
         $cmd .= "-var 'image_id=" . $image . "' " if ($image);
+        if (is_azure) {
```

Comment:
Why is this condition needed?  In `terraform_destroy` you are only doing:
`$cmd .= " -var 'storage-account=$storage_account'" if ($storage_account);` without checking if it's azure or not.

Reply:
Because passing the `storage_account` variable is only valid for the Azure Terraform profile because only there the variable is present.

Reply:
But it is there already, see https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/e8e4e17ad27f7ee4482c71d326fec96923ddd544/lib/publiccloud/provider.pm#L498 or what am I missing?

Reply:
sorry, I mean `if (is_azure) {` but it is there, I needed to expand the code to see it (facepalm)